deeppavlov/core/trainers/utils.py

```diff
@@ -15,6 +15,7 @@
 from collections import OrderedDict, namedtuple
 from json import JSONEncoder
 from typing import List, Tuple, Union, Iterable
+from functools import partial
```

Comment:
```suggestion
from functools import partial
from json import JSONEncoder
from typing import List, Tuple, Union, Iterable
```
Imports should be in alphabetic order. Pycharm has command to sort them: alt+ctr+o. VSC should have something similar as well.